tests/js/server/aql/aql-geo.inc

```diff
@@ -216,916 +110,889 @@
-              }
-            }
+          meta['links'][withView[basename].name()] = {
+            fields: {geo: {analyzers: [testAnalyzers[basename].name()]}}
           };
         }
-        view = db._createView(viewName, "arangosearch", meta);
+        plainView[basename] = db._createView(basename + '_' + viewName, "arangosearch", meta);
+      }
+    }
+    function insertAll(name, theAnalyzer, x) {
+      if (!noIndex.hasOwnProperty(name)) {
+        createCollectionSet(name, theAnalyzer);
       }
-    },
 
-////////////////////////////////////////////////////////////////////////////////
-/// @brief tear down all
-////////////////////////////////////////////////////////////////////////////////
+      for (let i = 0; i < x.length; ++i) {
+        x[i]._key = "K" + (++keyCounter);
+      }
+      noIndex[name].insert(x);
+      withIndex[name].insert(x);
+      withView[name].insert(x);
+    }
+
+    function compareKeyLists(namea, a, nameb, b) {
+      let good = true;
+      let msg = "";
+      if (a.length !== b.length) {
+        good = false;
+        msg += "Different number of results " + namea + " (" + a.length +
+          ") and " + nameb + " (" + b.length + ").\n";
+      }
+      let i = 0;
+      let j = 0;
+      let count = 0;
+      while (i < a.length && j < b.length) {
+        if (a[i] < b[j]) {
+          good = false;
+          msg += "Found key " + a[i] + " in '" + namea + "' but not in '"
+            + nameb + "'\n";
+          ++i;
+        } else if (a[i] > b[j]) {
+          good = false;
+          msg += "Found key " + b[j] + " in '" + nameb + "' but not in '"
+            + namea + "'\n";
+          ++j;
+        } else {
+          ++i;
+          ++j;
+        }
+        if (++count >= 10) {
+          return {good, msg};
+        }
+      }
+      while (i < a.length) {
+        good = false;
+        msg += "Found key " + a[i] + " in '" + namea + "' but not in '"
+          + nameb + "'\n";
+        ++i;
+        if (++count >= 10) {
+          return {good, msg};
+        }
+      }
+      while (j < b.length) {
+        good = false;
+        msg += "Found key " + b[j] + " in '" + nameb + "' but not in '"
+          + namea + "'\n";
+        ++j;
+        if (++count >= 10) {
+          return {good, msg};
+        }
+      }
+      return {good, msg};
+    }
 
-    tearDownAll: function () {
-      db._dropView(viewName);
-      db._drop(collWithoutIndex);
-      db._drop(collWithIndex);
-      db._drop(collWithView);
-      let analyzers = require("@arangodb/analyzers");
-      analyzers.remove("geo_json");
-    },
+    function compare(query, queryView) {
+      let fullScanQuery = `FOR d IN @@coll ${query} RETURN d._key`;
+      let viewQuery = `FOR d IN ${viewName} ${queryView} RETURN d._key`;
+      let invertedIndexQuery = `FOR d IN ${collWithIndex} OPTIONS {indexHint: "inverted", forceIndexHint: true, waitForSync: true} ${query} RETURN d._key`;
```

Comment:
In general will be better if you move it the setupAll, same as views.
Also it can be improved because search-alias view use inverted indexes (but it question for @alexbakharew why didn't he does it). Not necessary to create separate view and inverted index collection. Also not necessary to create double count of inverted indexes.
```suggestion
      let invertedIndexQuery = `FOR d IN ${collWithIndex} OPTIONS {indexHint: "inverted", forceIndexHint: true } ${query} RETURN d._key`;
```